File: Atividades Udemy/Java/secao07/atividade05.java
package secao07;

import java.util.Scanner; 

public class atividade05 {

	public static void main(String[] args) {
		String nome, senha;
		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Informe um nome de usuário: ");
		nome = teclado.next();
		System.out.println("Informe uma senha: ");
		senha = teclado.next();
		
		while(nome.equals(senha)) {
			System.out.println("Erro: o nome de usuário e a senha devem ser diferentes.");
			System.out.println("Informe um nome de usuário: ");
			nome = teclado.next();
			System.out.println("Informe uma senha: ");
			senha = teclado.next();
		}
		System.out.println("Parabéns! Você acessou o programa!");
		teclado.close();
	}

}

File: Atividades Udemy/Python/secao07/atividade05.py
nome = input("Informe o nome de usuário: ")
senha = input("Informe a senha: ")

while (nome == senha):
    print("O nome de usuário e senha devem ser diferentes.")
    nome = input("Informe o nome de usuário: ")
    senha = input("Informe a senha: ")
    
print("Você acessou o programa!")